apps/simple-trading-app/src/app/app.tsx

```diff
@@ -9,12 +9,14 @@ import {
   VegaManageDialog,
   VegaWalletProvider,
 } from '@vegaprotocol/wallet';
-import { DealTicketContainer } from './components/deal-ticket';
+// import { DealTicketContainer } from './components/deal-ticket';
```

Comment:
nitpick (non-blocking): this and below commented code can be deleted?

Reply:
I left it here as I think it will be helpful in conflict resolving, because we have here an early stage of the app and often have to write in the same files.